apps/frontend/src/api-util.ts

```diff
@@ -1,25 +1,15 @@
 import { API_BASE_URL } from './config';
 
 export type GetListOptions = {
-  searchQuery?: string;
-  filters?: string[];
-  currentPage?: number | null;
-  pageSize?: number | null;
-  sort?: {
-    sortBy: string; // column name ie startDate or endDate
-    sortOrder: 'asc' | 'desc';
-  };
+  searchQuery: string;
```

Comment:
This is the part about reducing duplicate 'nullish' states. most of the lines in the PR are a consequence of this essentially